src/BuildScriptGenerator/Node/NodeScriptGeneratorOptions.cs

```diff
@@ -7,15 +7,14 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Node
 {
     public class NodeScriptGeneratorOptions
     {
-        /// <summary>
-        /// Represents user provided version.
-        /// </summary>
-        public string NodeVersion { get; set; }
-
         /// <summary>
         /// Gets or sets the custom 'npm run build' command that is run after 'npm install' is run in the generated
         /// build script.
         /// </summary>
         public string CustomNpmRunBuildCommand { get; set; }
+
+        public bool PruneDevDependencies { get; set; }
+
+        public string NpmRegistryUrl { get; set; }
```

Comment:
It'll be great to have some descriptions over here.

Reply:
do we need some descriptions here?